components/authz-service/engine/opa/policy.bindata.go

```diff
@@ -1,18 +1,17 @@
-// Code generated by go-bindata.
+// Code generated by go-bindata. DO NOT EDIT.
```

Comment:
Hmm this could mean we're using different versions of `go-bindata`. That's kind of asking for trouble...

I'm on
```
$ go-bindata  -version
go-bindata 3.1.0 (Go runtime go1.11.3).
Copyright (c) 2010-2013, Jim Teeuwen.
```
and I've installed it using `go get`, I suppose. What version do you use? Let's standardize. 😉

Reply:
i'm on ```go-bindata 3.13.0 (Go runtime go1.11.4).
Copyright (c) 2010-2015, Jim Teeuwen.
Copyright (c) 2017-2018, Kevin Burke.```

Reply:
Yeah. Where did you get that? 😉

Reply:
Ah, https://github.com/kevinburke/go-bindata

Reply:
```
$ go get -u github.com/kevinburke/go-bindata/go-bindata
$ go-bindata  -version
go-bindata 3.13.0 (Go runtime go1.12.5).
Copyright (c) 2010-2015, Jim Teeuwen.
Copyright (c) 2017-2018, Kevin Burke.
$
```
:woohoo: